libethereum/ValidationSchemes.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+    This file is part of cpp-ethereum.
+
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "ValidationSchemes.h"
+#include <libdevcore/JsonUtils.h>
+#include <string>
+
+using namespace std;
+using namespace dev;
```

Comment:
You don't need this, everything is defined inside `dev` anyway.
You probably don't need `using namespace std;` either